### PR TITLE
doc: add MicroAppWithMemoHistory & connectMaster doc

### PR DESCRIPTION
**Pull request by brickspert**
##### Checklist

- [ ] `npm test` passes
- [ ] tests are included
- [x] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/umi/ISSUE_URL

-----
[View rendered docs/plugins/plugin-qiankun.zh-CN.md](https://github.com/umijs/umi/blob/doc/plugin-qiankun/docs/plugins/plugin-qiankun.zh-CN.md)

### CODE REVIEW

File: docs/plugins/plugin-qiankun.zh-CN.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ $ cd packages/plguin-qiankun && yarn start
 - ✔︎ 父子应用通讯
 - ✔︎ 子应用运行时配置自定义 `bootstrap()`、`mount()` 和 `unmount()`
 - ✔︎ 主应用、子应用联调
+- ✔︎ 子应用与子应用组合
```
Comment (kuitos) [inline docs/plugins/plugin-qiankun.zh-CN.md:39]:
```suggestion
- ✔︎ 嵌套子应用
```

File: docs/plugins/plugin-qiankun.zh-CN.md [hunk 2]
```diff
@@ -293,6 +306,45 @@ PORT=8081
 
 2. 子应用在生命周期钩子中获取 props 消费数据（参考子应用运行时配置一节）
 
+### 子应用与子应用嵌套
```
Comment (kuitos) [inline docs/plugins/plugin-qiankun.zh-CN.md:309]:
```suggestion
### 嵌套子应用
```

**Review by kuitos — CHANGES_REQUESTED** [review]

**Review by kuitos — APPROVED** [review]

File: docs/plugins/plugin-qiankun.md [hunk 3]
```diff
@@ -1,7 +1,3 @@
----
```
Comment (sorrycc) [inline docs/plugins/plugin-qiankun.md:1]: 这个咋删了？内容还是中文版的。

Comment (brickspert) [reply docs/plugins/plugin-qiankun.md:1]: 不小心删掉了，我补一下

**Pull request merged**